### PR TITLE
Use required classes for Lifecycle Callback examples

### PR DESCRIPTION
**Pull request by SerheyDolgushev**
Imports `Doctrine\ORM\Mapping\{Entity, HasLifecycleCallbacks, PrePersist, PreUpdate};` in Lifecycle Callback examples

### CODE REVIEW

File: docs/en/cookbook/validation-of-entities.rst [hunk 1]
```diff
@@ -58,6 +58,7 @@ First Attributes:
 .. code-block:: php
 
     <?php
+    use Doctrine\ORM\Mapping\{Entity, HasLifecycleCallbacks, PrePersist, PreUpdate};
```
Comment (SenseException) [inline docs/en/cookbook/validation-of-entities.rst]: To fit our coding style:

```suggestion
    use Doctrine\ORM\Mapping\Entity;
    use Doctrine\ORM\Mapping\HasLifecycleCallbacks;
    use Doctrine\ORM\Mapping\PrePersist;
    use Doctrine\ORM\Mapping\PreUpdate;
```

File: docs/en/reference/events.rst [hunk 2]
```diff
@@ -215,6 +215,7 @@ specific to a particular entity class's lifecycle.
         <?php
         use Doctrine\DBAL\Types\Types;
         use Doctrine\ORM\Event\PrePersistEventArgs;
+        use Doctrine\ORM\Mapping\{Entity, HasLifecycleCallbacks, PrePersist, PreUpdate};
```
Comment (SenseException) [inline docs/en/reference/events.rst]: To fit our coding style:

```suggestion
        use Doctrine\ORM\Mapping\Entity;
        use Doctrine\ORM\Mapping\HasLifecycleCallbacks;
        use Doctrine\ORM\Mapping\PrePersist;
        use Doctrine\ORM\Mapping\PreUpdate;
```

**Review by SenseException — CHANGES_REQUESTED** [review]

File: docs/en/cookbook/validation-of-entities.rst (hunk 1 above)
Comment (SerheyDolgushev) [reply docs/en/cookbook/validation-of-entities.rst]: Fixed in https://github.com/doctrine/orm/pull/10916/commits/d02b5bcf5d82762c54c4a48f0bac0857559e43ce

File: docs/en/reference/events.rst (hunk 2 above)
Comment (SerheyDolgushev) [reply docs/en/reference/events.rst]: Fixed in https://github.com/doctrine/orm/pull/10916/commits/d02b5bcf5d82762c54c4a48f0bac0857559e43ce

**Review by SenseException — APPROVED** [review]

**Comment by SenseException** [comment]
Thanks @SerheyDolgushev

**Pull request merged**